drivers/remote/api/api.go

```diff
@@ -16,6 +16,12 @@ func (r *Response) GetError() string {
 	return r.Err
 }
 
+// GetCapabilityResponse is the response of GetCapability request
+type GetCapabilityResponse struct {
+	Response
+	Scope string
```

Comment:
I just found that to make use of driver.call function, the GetCapabilityResponse need to contain `Response`, without this, I need to write some duplicate code as `driver.call`.
So I think maybe we'd better keep this although the error is not what we are interested in.